docs/faq.md

```diff
@@ -1,3 +1,5 @@
+> The official Splunk documentation for this page is [](). For instructions on how to contribute to the docs, see [CONTRIBUTING.md](../CONTRIBUTING#documentation.md).
```

Comment:
If there isn't a link should we remove that statement?

Reply:
Correct. Currently far from the computer, feel free to amend yourself!